homeassistant/components/freedompro/lock.py

```diff
@@ -75,10 +76,10 @@ async def async_added_to_hass(self) -> None:
         await super().async_added_to_hass()
         self._handle_coordinator_update()
 
-    async def async_lock(self, **kwargs):
+    async def async_lock(self, **kwargs: Any) -> None:
         """Async function to lock the lock."""
-        payload = {"lock": 1}
-        payload = json.dumps(payload)
+        payload_dict = {"lock": 1}
+        payload = json.dumps(payload_dict)
```

Comment:
I had to adjust this as mypy does not like reassigning a variable to a different type